### PR TITLE
Update to go1.16 and drop darwin/386 support

### PR DESCRIPTION
**Pull request by johejo**
darwin/386 is no longer supported since go1.15.
https://github.com/golang/go/issues/37610

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,16 +5,12 @@ services:
...
-      - ./docker/mysql/data:/var/lib/mysql
     ports:
       - '3306:3306'
   postgres:
     image: postgres:12
     environment:
       POSTGRES_USER: postgres
       POSTGRES_HOST_AUTH_METHOD: trust
-    volumes:
-      - ./docker/postgresql/data:/var/lib/postgresql/data
```
Comment (k0kubun) [inline docker-compose.yml:18]: Why did you remove it?

File: Makefile [hunk 2]
```diff
@@ -27,7 +27,6 @@ package:
 	$(MAKE) package-targz GOOS=linux GOARCH=arm64
 	$(MAKE) package-targz GOOS=linux GOARCH=arm
 	$(MAKE) package-zip GOOS=darwin GOARCH=amd64
-	$(MAKE) package-zip GOOS=darwin GOARCH=386
```
Comment (k0kubun) [inline Makefile:30]: Did Go stop supporting it?

File: docker-compose.yml (hunk 1 above)
Comment (johejo) [reply docker-compose.yml:18]: `go get -t ./...` fails in github actions
https://github.com/k0kubun/sqldef/runs/2302532462?check_suite_focus=true

File: Makefile (hunk 2 above)
Comment (johejo) [reply Makefile:30]: See PR description(edited)

**Review by k0kubun — APPROVED** [review]
